canvas_api_client/canvas_api_client/v1_client.py

```diff
@@ -169,41 +214,43 @@ def get_account_courses(self,
 
     def get_course_info(self,
                         course_id: str,
-                        is_sis_course_id: bool = False,
+                        is_sis_course_id: Optional[bool] = None,
                         params: RequestParams = None) -> Response:
         """
         Get the course information for a given course.
 
         https://canvas.instructure.com/doc/api/courses.html#method.courses.show
         """
-        if is_sis_course_id:
-            course_id = self._format_sis_course_id(course_id)
+        course_id = self._format_sis_course_id(course_id, is_sis_course_id)
 
         endpoint = "courses/{}".format(course_id)
 
         return self._get(self._get_url(endpoint), params=params)
 
     def get_course_users(self,
                          course_id: str,
-                         is_sis_course_id: bool = False,
+                         is_sis_course_id: Optional[bool] = None,
+                         flatten_response: Optional[bool] = None,
                          params: RequestParams = None) -> Iterator[Response]:
         """
         Returns a generator of course enrollments for a given course from the
         v1 Canvas API.
 
         https://canvas.instructure.com/doc/api/courses.html#method.courses.users
         """
-        if is_sis_course_id:
-            course_id = self._format_sis_course_id(course_id)
-
+        course_id = self._format_sis_course_id(course_id, is_sis_course_id)
         endpoint = "courses/{}/users".format(course_id)
 
+        flatten_response = flatten_response or self._flatten_response
```

Comment:
trivial change request: can we call this `is_flat` or something like it? i don't like that we do a redefinition of the `flatten_response` argument

Reply:
Sure!  I think I had this line in there as prelude to a debug statement, so I just merged it with the following line.

Reply:
looks good thanks!